aldryn_people/cms_toolbar.py

```diff
@@ -47,7 +47,7 @@ def get_obj_from_request(model, request,
 class PeopleToolbar(CMSToolbar):
     # watch_models must be a list, not a tuple
     # see https://github.com/divio/django-cms/issues/4135
-    watch_models = [Person, ]
+    watch_models = [Person, Group, ]
```

Comment:
Its not really, but I noticed during testing it wasn't there, and caused issues when re-slugging a group